src/openrct2/world/footpath.c

```diff
@@ -346,8 +347,19 @@ static money32 footpath_place_real(sint32 type, sint32 x, sint32 y, sint32 z, si
         return MONEY32_UNDEFINED;
     }
 
-    if (flags & GAME_COMMAND_FLAG_APPLY)
-        footpath_interrupt_peeps(x, y, z * 8);
+    if ((flags & GAME_COMMAND_FLAG_APPLY)) {
```

Comment:
I think it's an original bug. If you put down a path ghost, guests will often try to walk across it - I'd say the ghosts should be ignored under all circumstances.